content/javascript/concepts/dates/terms/getUTCMinutes/getUTCMinutes.md

```diff
@@ -0,0 +1,35 @@
+---
+Title: '.getUTCMinutes()'
+Description: 'Returns the minutes of the provided date in universal time.'
+Subjects:
+  - 'Web Development'
+Tags:
+  - 'Date'
+  - 'Methods'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`.getUTCMinutes()`** method returns the minutes for the date provided according to Coordinated Universal Time (UTC).
+
+## Syntax
+
+```pseudo
+myDate.getUTCMinutes()
+```
```

Comment:
[Nitpick] Since this is a JavaScript code snippet, remember to add the corresponding abbreviation next to it. See [the style guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#supported-code-blocks) for more context. 
```suggestion
```js 
myDate.getUTCMinutes()
```
```

Reply:
@CBID2 Actually, this will stay as "pseudo" since this is showing the syntax. The language is specified in examples. 😄

Reply:
Ohhh